docker/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+# This is a docker-compose file for use by Resource Providers
+services:
+  ipfs:
+    image: ipfs/kubo:v0.30.0
+    container_name: ipfs
+    restart: unless-stopped
+    ports:
+      - 4001:5001
+      - 127.0.0.1:5001:5001
+      - 127.0.0.1:8080:8080
+    volumes:
+      - ipfs-data:/data/ipfs
+  resource-provider:
+    image: ghcr.io/lilypad-tech/resource-provider:latest
+    container_name: resource-provider
+    restart: unless-stopped
+    depends_on:
+      - ipfs
+    build:
```

Comment:
If you're pulling from a registry, you might want to remove the build section to ensure everything runs smoothly